### ISSUE TITLE
Warning spam

### ISSUE THREAD

**Issue by ArturZahn** [appliedengdesign/vscode-gcode-syntax#60]
When I'm working on more than 10K lines files (all the time) the extension send an warning saying something may not work.

that's OK, I click in the "continue" button to continue working anyway, but the warning pops up every time I make any changes to the file, every time I type a character it pops up... It's annoying, I think it would be better if it showed the warning just once when I opened the file. 

![image](https://user-images.githubusercontent.com/62709763/218326551-b2b350df-7819-4564-bfb2-2e517d63a5c9.png)

**Comment by mikecentola** [comment]
Good catch. It should be a simple flag I can add into there to fix this. Sorry for taking so long to get back to this. I've been offline dealing with some life issues.
